src/lib.rs

```diff
@@ -14,34 +14,57 @@
 //!
 //! # Examples
 //!
+//! Get a channel
+//!
+//! ```rust,no_run
+//! use twitch_api::helix::HelixClient;
+//! use twitch_api::twitch_oauth2::{AccessToken, UserToken};
+//!
+//! #[tokio::main]
+//! async fn main() -> Result<(), Box<dyn std::error::Error + Send + Sync + 'static>> {
+//!     // Create the HelixClient, which is used to make requests to the Twitch API
```

Comment:
this comment made rustfmt not format this as

```rs
//! #[tokio::main]
//! async fn main(
//! ) -> Result<(), Box<dyn std::error::Error + Send + Sync + 'static>> {
```

strange

Reply:
ah, it breaks rustfmt, nice

Reply:
Maybe that's a rustfmt bug 🙃.

Reply:
probably https://github.com/rust-lang/rustfmt/issues/5627 due to https://github.com/twitch-rs/twitch_api/blob/38dca416db2707b85e35bb582c8495e0cc8a60c4/rustfmt.toml#L10